packages/salesforcedx-utils-vscode/src/services/sourceTrackingService.ts

```diff
@@ -38,7 +38,6 @@ export class SourceTrackingService {
     const options: SourceTrackingOptions = {
       org,
       project,
-      ignoreLocalCache: true,
```

Comment:
My hunch is that this will also have an effect on the accuracy of calls to "View Changes" commands.  IIRC this was here because without it STL would get out of sync when creating new files locally/on the server, and STL doesn't expose a command to "refresh" or "re-index" it on demand.  Will investigate.